Communitheme/gtk-3.0/_colors.scss

```diff
@@ -80,3 +80,7 @@ $hb_pathbar_bg: lighten($headerbar_bg_color, 5%);
 $hb_pathbar_bg_backdrop: mix(lighten($headerbar_bg_color, 5%), $headerbar_bg_color, 50%);
 $hb_pathbar_border_backdrop: darken($hb_pathbar_bg_backdrop, 2%);
 $hb_button_bg_hover: lighten($headerbar_bg_color, 10%);
+
+//box-shadows
+$window_decoration_shadow: 0 3px 9px 1px transparentize(black, 0.5);
```

Comment:
I do not see the reason to create a variable for an expression which is used only once

Reply:
Oh it should beused twice. For the app notifications too. Pushing this later.

Shell notifications, gtk windows, gtk app notifications  all have the same shadow. Shell is ofc not in here ;0 but this way it is very clear that it is not some random box shadow but rather one specific shadow. 
Same for popovers and context menu

Reply:
Right and I agree, but I still see no reason to use same variable names to link different projects ;)

Reply:
In the shell it has no name (yet)

Reply:
@clobrano is this okay now?